.travis.yml

```diff
@@ -64,12 +53,10 @@ before_install:
 install:
   - "travis_retry pip install -r .travis-${REQUIREMENTS}-requirements.txt"
   - "travis_retry pip install -e .[$EXTRAS]"
+  - "travis_retry pip install git+https://github.com/ppanero/docker-services.git#egg=docker-services"
```

Comment:
This would be a PyPI package, right?

Reply:
Yes

Reply:
yes, this is here until we actually release it. just for demo